pkg/network/protocols/kafka/protocol.go

```diff
@@ -40,10 +40,12 @@ const (
 	eventStreamName = "kafka"
 	filterTailCall  = "socket__kafka_filter"
 
-	responsePartitionParserV0TailCall    = "socket__kafka_response_partition_parser_v0"
-	responsePartitionParserV12TailCall   = "socket__kafka_response_partition_parser_v12"
-	responseRecordBatchParserV0TailCall  = "socket__kafka_response_record_batch_parser_v0"
-	responseRecordBatchParserV12TailCall = "socket__kafka_response_record_batch_parser_v12"
+	responsePartitionParserV0TailCall        = "socket__kafka_response_partition_parser_v0"
```

Comment:
We could rename these to `_produce_` later (in a separate PR) to avoid confusion.

Reply:
wdym? these are for fetch

Reply:
I'll make the renaming as part of this PR

Reply:
On second thought, I'll open another PR for it, to keep this PR simpler

Reply:
It turned out to be lots of changes, so I'll do it in a separate PR to keep this one simpler:
https://datadoghq.atlassian.net/browse/USMON-1210